### PR TITLE
Add Windows11 default font configuration. fixes #488

### LINKED ISSUE #488: Windows11: Missing Font-Properties-File in Bundle org.eclipse.jface
There is no Default-Font-Properties-File in Bunlde org.eclipse.jface for Windows 11

org.eclipse.jface_3.28.0.v20221024-1426\org\eclipse\jface\resource\jfacefonts_**windows11**.properties

As a result the default-fonts look outdated under Windows11 due to the fallbacks in FontRegistry#readResourceBundle.

### PR DESCRIPTION
**Pull request by Rainer7000**
Issue:
https://github.com/eclipse-platform/eclipse.platform.ui/issues/488

### CODE REVIEW

**Comment by akurtakov** [comment]
Please sign ECA or otherwise your patch can't be incorporated.  https://accounts.eclipse.org/legal/eca/validation/153038

File: bundles/org.eclipse.jface/src/org/eclipse/jface/resource/jfacefonts_windows11.properties [hunk 1]
```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2015 IBM Corporation and others.
```
Comment (jukzi) [inline bundles/org.eclipse.jface/src/org/eclipse/jface/resource/jfacefonts_windows11.properties]:
```suggestion
# Copyright (c) 2023 IBM Corporation and others.
```

**Comment by iloveeclipse** [comment]
@Rainer7000 : thanks for patches. Please
1) Avoid merge commits - instead, **rebase** your changes on top of master.
2) Create one commit per PR - so please rebase your changes and squash into one commit
3) Update commit message, see https://github.com/eclipse-platform/.github/blob/main/CONTRIBUTING.md#commit-message-recommendations

**Comment by jukzi** [comment]
please squash changes to a single commit.

**Comment by Rainer7000** [comment]
Is there anyone from the platform-team who can help me commit this patch please ? 
It's only a simple ascii-file with some default-fonts and the commit ist stuck for months now on "Code Analysis Queued — Waiting to run this check...".

**Comment by vogella** [comment]
We are currently in a freeze period so you will have to wait 1-2 weeks more.

**Comment by vogella** [comment]
Sorry.

**Comment by Rainer7000** [comment]
For comparison here are screeshots from before the patch, where Jface defaults to Courier new Font for Code or the Default-Text-Font:
before

After the patch the default Text-Font under Windows11 is Consolas  (same as under Windows 10):
after

**Comment by vogella** [comment]
Thanks @Rainer7000 finally tested on Win11, looks OK to me (not better, not worse for me but as a rare Window user I'm not used to the "right" font in Windows).

@Rainer7000 can you verify in the next successful I-build (currently we are having problems to get one, most likely due to https://github.com/eclipse-jdt/eclipse.jdt.core/issues/1394) ?  https://download.eclipse.org/eclipse/downloads/

**Comment by Rainer7000** [comment]
I have successfully checked the expected default jface-fonts under Windows 11 in Build I20231218-1800.

**Comment by vogella** [comment]
Thanks @Rainer7000

**Pull request merged**